snap/README.md

```diff
@@ -447,21 +447,21 @@ Example: `snap set edgexfoundry env.device-virtual.service.port=7777`
 
 | API Gateway setting | Description                                                  |
 | ------------------- | ------------------------------------------------------------ |
-| add-proxy-route     | Defines a configuration TOML containing a list of URLs to be added to the API Gateway (aka Kong). See [documentation](https://docs.edgexfoundry.org/2.1/security/Ch-APIGateway/#configuration-of-adding-microservices-routes-for-api-gateway) NOTE - this setting is not a configuration override, it's a top-level environment variable used by the security-proxy-setup. That means it can not be configureed thourgh `snap set` |
+| add-proxy-route     | Defines a configuration TOML file containing a comma-separated list of paired additional service names and URLs to create proxy Kong routes. See [documentation](https://docs.edgexfoundry.org/2.1/security/Ch-APIGateway/#configuration-of-adding-microservices-routes-for-api-gateway) NOTE - this setting is not a configuration override, which means it can not be configured through `snap set`. it's a top-level environment variable used by the security-proxy-setup. |
```

Comment:
I'm not sure why this is called a "configuration TOML file". From the linked document, this is a URL: "myApp.http://my-app:56789" and multiple URLs can be passed as a comma-separated list.

Reply:
It was not right. I have updated it. Thanks!